Quick_Deploy/vLLM/README.md

```diff
@@ -55,7 +55,7 @@ required to serve the [facebook/opt-125m](https://huggingface.co/facebook/opt-12
 ```
 mkdir -p model_repository/vllm_model/1
 wget -P model_repository/vllm_model/1 https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/samples/model_repository/vllm_model/1/model.json
```

Comment:
May I kindly ask you to change `main` here as well, and add the following after the code block:
"where <xx.yy> is the version of Triton that you want to use. Please note, that Triton's vLLM container has been introduced starting from 23.10 release."

Reply:
Happy to approve after. I believe for this simple change CLA is not required

Reply:
sure! done (no problem signing the CLA)

Reply:
For CLA, I just need a signed copy of the [Contributor License Agreement](https://github.com/NVIDIA/triton-inference-server/blob/master/Triton-CCLA-v1.pdf) to [triton-cla@nvidia.com](mailto:triton-cla@nvidia.com). NOTE: Contributors with no company affiliation can fill N/A in the Corporation Name and Corporation Address fields.

Reply:
Done!